openhtf/io/output/mfg_inspector.py

```diff
@@ -232,10 +231,10 @@ def _ExtractParameters(record, testrun, used_parameter_names):
       if measurement.units:
         testrun_param.unit_code = UOM_CODE_MAP[measurement.units.uom_code]
 
-      if name not in phase.measured_values:
+      if not measurement.measured_value:
         testrun_param.status = test_runs_pb2.ERROR
         continue
-      value = phase.measured_values[name]
+      value = measurement.measured_value
```

Comment:
if `dimensions` is `None`, then isn't `measurement.measured_value` a `measurements.MeasuredValue` that you still have to get the `.value` of?  Same for dimensioned measurements, I see a lot of places where `value` is used directly, but I suspect it needs to be `value.value`, such as when passed to `_MangleMeasurement()`.  Aliasing `measured_value` to `value` is confusing, too.